src/components/PeopleItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  human: Person,
+  people: Person[],
+};
+
+export const PeopleItem: React.FC<Props> = ({
+  human,
+  people,
+}) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother = people.find(pers => pers.name === motherName),
+    father = people.find(pers => pers.name === fatherName),
+  } = human;
+
+  const { slugName } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': slug === slugName })}
+    >
+      <td>
+        <Link
+          to={`../${slug}`}
```

Comment:
try to fix it:
![image](https://github.com/mate-academy/react_people-table-advanced/assets/86923089/9b8fe69e-dae9-4cb2-9147-3e422878d281)

Reply:
if you don't know how to fix that, you can ask this question in a fe chat